tests/tick.test.mjs

```diff
@@ -415,8 +415,8 @@ describe('Tick', async () => {
     });
 
     // get the epoch timestamp and assert it is in 24 hours
-    const protocolBalanceAtStartOfEpoch = 50_000_000_0000; // 50M ARIO
-    const totalEligibleRewards = protocolBalanceAtStartOfEpoch * 0.05; // 5% of the protocol balance
+    const protocolBalanceAtStartOfEpoch = 50_000_000_000_000; // 50M ARIO
+    const totalEligibleRewards = protocolBalanceAtStartOfEpoch * 0.001; // 0.1% of the protocol balance
```

Comment:
this expectation was actually 0.05% of the balance, but the 50M ARIO value was wrong. so the `// 50M ARIO` comment was correct, but the `//  0.1% of of the protocol balance` and the mARIO number value was incorrect